src/installing-workflow/configuring-registry.md

```diff
@@ -0,0 +1,67 @@
+# Configuring Registry
+
+Deis Workflow's builder component relies on registry for storing the application docker images.
+
+Deis Workflow ships with [registry][registry] component by default, which provides in-cluster docker registry backed by the platform configured [object storage][storage]. Operators might want to use off-cluster registry mainly for performance or security reasons.
```

Comment:
I would add "a" before "registry," "an" before "in-cluster," capitalize Docker, hyphenate "platform-configured," drop "mainly," and add "an" before "off-cluster":

> Deis Workflow ships with a [registry][registry] component by default, which provides an in-cluster Docker registry backed by the platform-configured [object storage][storage]. Operators might want to use an off-cluster registry for performance or security reasons.